SoftwareGuide/Latex/Introduction/Installation.tex

```diff
@@ -589,6 +652,73 @@ \section{Getting Started With ITK}
 to see the output. It is therefore preferable to run the executable from the DOS
 command line by starting the \code{cmd.exe} shell first.
 
+\section{Using ITK as an External Library}
+\label{sec:UsingITKAsExternalLibrary}
+
+For a project that uses ITK as an external library, it is recommended to
+specify the individual ITK modules in the \code{COMPONENTS} argument in the
+\code{find_package} CMake command:
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+find_package(ITK REQUIRED COMPONENTS Module1 Module2)
+include(\${ITK_USE_FILE})
+\end{minted}
+\normalsize
+
+e.g.
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+find_package(ITK REQUIRED
+  COMPONENTS
+    MorphologicalContourInterpolation
+    ITKSmoothing
+    ITKIOImageBase
+    ITKIONRRD
+    )
+include(\${ITK_USE_FILE})
+\end{minted}
+\normalsize
+
+If you would like to use the CMake ExternalProject
+Module\footnote{url{https://cmake.org/cmake/help/latest/module/ExternalProject.html}}
+to download ITK source code when building your ITK application (a.k.a.
+Superbuild ITK), here is a basic CMake snippet for setting up a Superbuild in
+an ITK application project using CMake:
+
+\small
+\begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{bash}
+ExternalProject_Add(ITK
+      GIT_REPOSITORY \${git_protocol}://github.com/InsightSoftwareConsortium/ITK.git"
+      GIT_TAG "<tag id>" # specify the commit id or the tag id
+      SOURCE_DIR <ITK source tree path>
+      BINARY_DIR <ITK build tree path>
+      CMAKE_GENERATOR ${gen}
+      CMAKE_ARGS
+          ${ep_common_args}
+          -DBUILD_SHARED_LIBS:BOOL=OFF
+          -DBUILD_EXAMPLES:BOOL=OFF
+          -DBUILD_TESTING:BOOL=OFF
+          -DITK_BUILD_DEFAULT_MODULES:BOOL=ON
+          -DModule_ITKReview:BOOL=ON
```

Comment:
We should remove this to discourage folks from using `ITKReview`.

Reply:
Done.